src/foam/u2/view/ScrollTableView.js

```diff
@@ -348,12 +348,7 @@
           var page = this.currentTopPage_ + i;
           if ( this.renderedPages_[page] ) continue;
           var dao = this.data$proxy.limit(this.pageSize).skip(page * this.pageSize);
-          var tbody = this.table_.slotE_(this.table_.rowsFrom(dao));
```

Comment:
on order changed there is no way for tbody div to know top position at which it should be